### PR TITLE
Fix #2463: remove textdirection property from textInputLayout

### LINKED ISSUE #2463: Optimise TextInputLayout in profile_reset_pin_activity
Remove `android:textDirection="locale"` in `TextInputLayout` from the following files
 - layout/profile_reset_pin_activity
 - layout-land/profile_reset_pin_activity

### PR DESCRIPTION
**Pull request by alwin48**
## Explanation

Fixes #2463: Removed android:textDirection='locale' in TextInputLayout from layout/profile_reset_pin_activity and layout-land/profile_reset_pin_activity

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -20,7 +20,8 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
+        <option name="useQualifiedModuleNames" value="true" />
```
Comment (FareesHussain) [inline .idea/gradle.xml]: Please revert this

**Review by FareesHussain — CHANGES_REQUESTED** [review]
PTAL

File: .idea/gradle.xml (hunk 1 above)
Comment (alwin48) [reply .idea/gradle.xml]: Okay, I will do that.

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

**Comment by anandwana001** [comment]
@alwin48 please correct the pr title as per the first point in the checklist.

**Comment by alwin48** [comment]
> @alwin48 please correct the pr title as per the first point in the checklist.

Done.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**